workflow/automation/templates/adv_im_calc.sl.template

```diff
@@ -33,30 +33,30 @@ end_time=`date +$runtime_fmt`
 # verify the result of the analysis
 fd_name=$(getFromYaml "{{sim_dir}}/sim_params.yaml" FD_STATLIST)
 echo "=========="
-python $IMPATH/Advanced_IM/scripts/check_adv_IM_status.py {{sim_IM_calc_dir}} {{models}} --stations {{station_file}}; success=$?
+python $IMPATH/../Advanced_IM/scripts/check_adv_IM_status.py {{sim_IM_calc_dir}} {{models}} --stations {{station_file}}; success=$?
```

Comment:
I'm not a huge fan of the ../ in the filepath when we've created a variable above? Would it be better to have a separate variable?